thirdparty/cmdconfig/commands/cmdsink/cmdsink.go

```diff
@@ -35,7 +34,11 @@ type SinkRunner struct {
 }
 
 func (r *SinkRunner) runE(c *cobra.Command, args []string) error {
-	outputs := []kio.Writer{&kio.LocalPackageWriter{PackagePath: args[0]}}
+	dir := "."
```

Comment:
do you assume POSIX?

Reply:
I believe this should also work on Windows. I notice that in other commands [some](https://github.com/GoogleContainerTools/kpt/blob/next/internal/cmdinit/cmdinit.go#L72) are using `pkg.CurDir` which is basically a `.` and [some](https://github.com/GoogleContainerTools/kpt/blob/next/internal/cmdliveinit/cmdliveinit.go#L84) are using `os.Getwd`.

Reply:
Should be consistent. Let's standardize on pkg.CurDir across the board.

Reply:
done